### DISCUSSION TITLE
Request for the demo app in November's webinar

### DISCUSSION THREAD

**Discussion by adarliu** [AzureAD/microsoft-authentication-library-for-android#1750] [Q&A]
Hi MSAL developers,
Is it possible to share the demo app codes in the [webinar ](https://www.youtube.com/watch?v=x1_snVp0rIw)? It looks quite different from the [sample project](https://github.com/Azure-Samples/ms-identity-android-java) in the official doc pages. And it looks much better to showcase more features in MSAL. Thanks!

**Comment by negoe** [comment]
Hi @adarliu, Thanks for the feedback. The demo app is our internal MSAL Android Test application. I will check on my end what can I do provide you more details around it. Thank you.
